readme.md

```diff
@@ -148,13 +148,17 @@ Default: `true`
 
 Automatically show the help text when the `--help` flag is present. Useful to set this value to `false` when a CLI manages child CLIs with their own help text.
 
+This option will only enabled when there is only one argument in `process.argv`.
```

Comment:
This needs to be added to https://github.com/sindresorhus/meow/blob/master/index.d.ts too